packages/valory/connections/ledger/ledger_dispatcher.py

```diff
@@ -338,6 +343,37 @@ async def get_transaction_receipt(
             )
         return response
 
+    @staticmethod
+    def _get_transaction_digest(
+        api: LedgerApi, message: LedgerApiMessage
+    ) -> Optional[str]:
+        """Send a signed transaction and get the transaction digest."""
+        if message.rpc_config and message.rpc_config.body.get(
+            "use_flashbots", DEFAULT_USE_FLASHBOTS
+        ):
+            target_blocks = message.rpc_config.body.get("target_block_numbers", None)
+            if target_blocks is None:
+                current_block_number = int(api.api.eth.get_block_number())
+                target_blocks = [
+                    *range(
+                        current_block_number,
+                        current_block_number + DEFAULT_NEXT_TARGET_BLOCKS,
+                    )
+                ]
+            target_blocks = cast(List[int], target_blocks)
```

Comment:
Ok, this is messy. I propose:

`if api.is_bundle(message)` and then move that custom logic into the method. That way, we remove any mention of "flashbots" here, which is not appropriate given the concerns at this layer. Second, we allow for each plugin to provide its own custom implementation of checking for transaction bundles.